validator/db/kv/attestation_history.go

```diff
@@ -35,6 +37,21 @@ func (store *Store) AttestationHistoryForPubKeys(
 			} else {
 				attestationHistory = enc
 			}
+			var minAtt attHist.MinAttestation
+			var setMin bool
+			enc = bucket.Get(attHist.GetMinSourceKey(key))
```

Comment:
Naming here is confusing, on one hand we have bucket keys and also public keys, let's refactor this better